app/Http/Controllers/Bible/VideoStreamController.php

```diff
@@ -93,12 +93,14 @@ public function jesusFilmsLanguages()
             }
 
             $cache_params = [$metadata_tag];
-            $languages = cacheRemember('arclight_languages_detail', $cache_params, now()->addDay(), function () use ($metadata_tag) {
+            $languages = cacheRemember('arclight_languages_detail_remove', $cache_params, now()->addHours(0), function () use ($metadata_tag) {
```

Comment:
I think that `now()->addHours(0)` should be replaced by  `now()->addDay()`, right?

Reply:
solved